### PR TITLE
Tweak sentence

### PR DESCRIPTION
**Pull request by clausmullie**
null

-----
[View rendered activities/codebase-stewardship/index.md](https://github.com/publiccodenet/about/blob/clausmullie-patch-1/activities/codebase-stewardship/index.md)

### CODE REVIEW

File: activities/codebase-stewardship/index.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ We can provide codebase stewardship from the beginning of the development or for
 
 ## What codebase stewardship is
 
-To explain codebase stewardship we split its activities in the categories
+To explain codebase stewardship it can be useful to consider perspectives from
```
Comment (Ainali) [inline activities/codebase-stewardship/index.md:26]:
```suggestion
To understand codebase stewardship it can be useful to consider perspectives from
```

Comment (Ainali) [reply activities/codebase-stewardship/index.md:26]: I think changing it from explaining to understanding might be even better.

**Review by Ainali — APPROVED** [review]
Yes, a good step in the right direction!

**Review by ericherman — COMMENTED** [review]
much better

**Pull request merged**